apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.cs

```diff
@@ -93,11 +161,8 @@ public static UrlSigner FromServiceAccountData(Stream credentialData)
         /// Creates a new <see cref="UrlSigner"/> instance for a service account.
         /// </summary>
         /// <param name="credential">The credential for the a service account. Must not be null.</param>
-        public static UrlSigner FromServiceAccountCredential(ServiceAccountCredential credential)
-        {
-            GaxPreconditions.CheckNotNull(credential, nameof(credential));
-            return new UrlSigner(new ServiceAccountCredentialBlobSigner(credential), SystemClock.Instance);
-        }
+        // TODO: Deprecate this method in favour of the new one.
```

Comment:
Note that if you want to leave the TODO and deprecate it later, the comment will need to move.